rust/azure_iot_operations_protocol/src/rpc_command/executor.rs

```diff
@@ -214,6 +214,40 @@ impl<TResp: PayloadSerialize> ResponseBuilder<TResp> {
     }
 }
 
+/// Helper function to add user-defined application error headers to a [`Vec<(String, String)>`] to be used as the `custom_user_data` on the [`Response`].
```

Comment:
this all looks fine to me, but run a quick `cargo doc --no-deps --all-features` if you haven't yet to make sure the docs generate fine and everything looks alright